tests/unit/index.ts

```diff
@@ -60,6 +61,44 @@ registerSuite({
 			assert.notInclude(contents, `module 'foo/index'`);
 		});
 	},
+	'project with outDir and rootDir - directory handling stress test': function () {
```

Comment:
Hmmm, that is odd... I'm not sure why GitHub thinks this will merge cleanly in this case.